### ISSUE TITLE
Can it analyze binary programs?

### ISSUE THREAD

**Issue by TienstinL** [michaekang/android_s2e#1]
I had watched your video about Symbolic Execution and Android_S2E. The example you provided is a C-program. But I need to analyze the binary programs. Please tell me anything else I can do to prepare for this? Thanks!

**Comment by michaekang** [comment]
Yes of course. That is just what S2E 'purpose is. To execute and analysis binary.
